Description: package & relax deps for tracing-durations-export
Author: eevelweezel <eevel.weezel@gmail.com>
Last-Update: 2025-01-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -60,3 +60,3 @@
 [dependencies.fs]
-version = "2.11.0"
+version = "3"
 package = "fs-err"
@@ -82,3 +82,3 @@
 [dependencies.svg]
-version = "0.17.0"
+version = "0.16"
 optional = true
